### PR TITLE
E2E: Describe BGP network topology

### PR DESCRIPTION
**Pull request by zeeke**
As discussed in #1204 , it's useful to have a diagram with the test network topology.

I put it in `e2etest/README.md` and double linked to the implementing code.

### CODE REVIEW

**Comment by oribon** [comment]
nice!

File: e2etest/README.md [hunk 1]
```diff
@@ -26,3 +27,39 @@ inv e2etest --focus L2
 The test suite will run the appropriate tests against the cluster.
 Be sure to cleanup any previously created development clusters using `inv dev-env-cleanup`.
 
+
+## BGP tests network topology
+
+For BGP test suite, a couple of BGP peers are needed to verify test scenario. The following
```
Comment (fedepaol) [inline e2etest/README.md:33]:
```suggestion
In order to test the multiple scenarios required for BGP, three different containers are required. The following
```

File: e2etest/README.md [hunk 2]
```diff
@@ -26,3 +27,39 @@ inv e2etest --focus L2
...
+diagram describe the networks, the pods and the containers involved.
```
Comment (fedepaol) [inline e2etest/README.md:34]:
```suggestion
diagram describes the networks, the pods and the containers involved.
```

File: e2etest/README.md [hunk 3]
```diff
@@ -26,3 +27,39 @@ inv e2etest --focus L2
...
+  │                    │                       │       │                │ │
+  └────────────────────┼───────────────────────┘       │ ebgp-multi-hop │ │
+                       │                               │                │ │
+                       │       multi-hop-net           └────────────────┘ │
+                       │       172.30.0.0/16                              │
+                       │  fc00:f853:ccd:e798::/64                         │
+                       └──────────────────────────────────────────────────┘
+```
+
+The above diagram is implemented in `bgptests.InfraSetup(...)`.
```
Comment (fedepaol) [inline e2etest/README.md:65]: Not sure we want to add a link to the code, but mention the file.

**Comment by fedepaol** [comment]
LGTM

**Pull request merged**